edflow/custom_logging.py

```diff
@@ -109,6 +109,8 @@ def init(
                 cls.resumed = False
                 cls.root = os.path.join(log_dir, name)
             else:
+                if run_dir[-1] == "/":
```

Comment:
This is the same as 
`run_dir = os.path.dirname(run_dir)`
which is potentially more robust.